Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_7_0.md

```diff
@@ -1,7 +1,5 @@
-
-
 #### Playbooks
 ##### New: Microsoft Defender For Endpoint - Unisolate Endpoint
-- This playbook will auto unisolate endpoints through Microsoft Defender For Endpoint by using hostnames, IPs, or Device IDs associated with the asset you wish to block (Available from Cortex XSOAR 6.2.0).
+This playbook auto unisolates endpoints with **Microsoft Defender For Endpoint** by using the host name, IP, or device ID associated with the asset you want to block (Available from Cortex XSOAR 6.2.0).
```

Comment:
@julieschwartz18 - the 'block' at the end should be changed to 'unblock'
Not sure also about the 'auto' term at the beginning. May I suggest 'This playbook will unisolate  **Microsoft Defender For endpoint** devices '  .....